--- origsrc/setuptools-60.2.0/setuptools/_distutils/spawn.py	2021-12-29 22:29:53.000000000 +0100
+++ src/setuptools-60.2.0/setuptools/_distutils/spawn.py	2022-01-03 00:18:48.595794900 +0100
@@ -35,7 +35,6 @@ def spawn(cmd, search_path=1, verbose=0,
     # in, protect our %-formatting code against horrible death
     cmd = list(cmd)
 
-    log.info(subprocess.list2cmdline(cmd))
     if dry_run:
         return
 
--- origsrc/setuptools-60.2.0/setuptools/_distutils/unixccompiler.py	2021-12-29 22:29:53.000000000 +0100
+++ src/setuptools-60.2.0/setuptools/_distutils/unixccompiler.py	2022-01-10 03:53:12.708338000 +0100
@@ -81,6 +81,7 @@ class UnixCCompiler(CCompiler):
     xcode_stub_lib_format = dylib_lib_format
     if sys.platform == "cygwin":
         exe_extension = ".exe"
+        dylib_lib_extension = ".dll.a"
 
     def preprocess(
         self,
